loop/routes/rooms.js

```diff
@@ -385,17 +385,22 @@ module.exports = function (apiRouter, conf, logError, storage, filestorage, auth
                   participants) {
                     if (res.serverError(err)) return;
 
+                    // The room participants list except this Hawk session.
```

Comment:
I'm expecting a comment to be more descriptive of what's hapenning here: why are we building this list. Also, the list of participants, passed to the `canJoinRoom` function is supposed to be the list of participants in the room, so if filtering there is, it probably should happen in there rather than before, otherwise, the passed list of participants actually isn't representative anymore of who's connected at a specific time (because it has been altered).